clang/lib/CodeGen/CGCall.cpp

```diff
@@ -1836,6 +1837,50 @@ static bool HasStrictReturn(const CodeGenModule &Module, QualType RetTy,
          Module.getLangOpts().Sanitize.has(SanitizerKind::Return);
 }
 
+llvm::fp::FPAccuracy convertFPAccuracy(StringRef FPAccuracy) {
```

Comment:
`static`?

Reply:
Also, why do we need to do this dance using strings instead of `FPAccuracyKind` enumeration values?

Reply:
This is specific to function. So we need to find if the accuracy for each function is needed. Unless I am not understanding your comment?
What would we do for a run command like this one:
 "-ffp-builtin-accuracy=high:[acosf,cos,pow] low:[tan] medium:[sincos,log10]" 
if we used the FPAccuracyKind.

Reply:
Removed the comment :) Responded to the wrong comment.

Reply:
Not sure what you mean here? The input the function is a string, and we are returning an FPAccuracyKind.